packages/gatsby-recipes/src/providers/utils/get-diff.js

```diff
@@ -12,7 +13,11 @@ module.exports = async (oldVal, newVal) => {
     expand: false,
   }
 
-  const diffText = diff(oldVal, newVal, options)
+  let diffText = diff(oldVal, newVal, options)
+
+  if (process.env.GATSBY_RECIPES_NO_COLOR) {
```

Comment:
i'm wondering if we could do this in the test instead, and not in code? Or do we want people to have this option too?

Reply:
like pass it in as an option into the function? That wouldn't keep it any more hidden.